docs/docs/guides/integrations/notdiamond.md

```diff
@@ -68,7 +68,6 @@ preference_id = train_router(
     response_column="actual",
     language="en",
     maximize=True,
-    api_key=api_key,
```

Comment:
This isn't the OpenAI constructor like the others, but it does appear to be an optional parameter.

Reply:
Agreed, that's why I removed it - seemed like it would be promoting the antipattern